tests/Models/UserModelTest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+use PHPUnit\Framework\TestCase;
+use Portal\Models\UserModel;
+
+class UserModelTest extends TestCase
+{
+    function testConstruct()
+    {
+            $db = new PDO('mysql:host=192.168.20.20;dbname=academyPortal', 'root');
```

Comment:
as above, mock the PDO (dont created a new connection)

Reply:
as above